scripts/dumps/bulk_genome_dump_from_core.py

```diff
@@ -0,0 +1,343 @@
+        query (str): The query to be executed.
+
+    Returns:
+        str: The output of the subprocess call to query the database.
+    """
+    mysql_call = [host, core_db, "-N", "-e", query]
+    return subprocess_call(command=mysql_call)
+
+
+def parse_yaml(file, dest):
+    """
+    YAML parser.
+
+    Args:
+        file (file object): The file object.
+        dest (str): The destination PATH where the FASTA file will be stored.
+    """
+    content = yaml.safe_load(file)
+    download_content = []
+    for data in content:
+        host = data["host"]
+        port = data["port"]
+        include_gca_number = data["gca_number"]
+
+        for core_db in data["core_db"]:
+            fasta_file_name = query_coredb(
+                host=host,
+                core_db=core_db,
+                query="SELECT meta_value FROM meta WHERE meta_key='species.production_name';",
+            )
+
+            if include_gca_number:
+                gca_number = query_coredb(
+                    host=host,
+                    core_db=core_db,
+                    query="SELECT meta_value FROM meta WHERE meta_key='assembly.accession';",
+                )
+                gca_number = gca_number.replace(".", "v").replace("_", "").lower()
+                fasta_file_name = f"{fasta_file_name}_{gca_number}"
+
+            # Query the genome components and split the result
+            genome_components = [
+                component
+                for component in query_coredb(
+                    host=host,
+                    core_db=core_db,
+                    query=(
+                        "SELECT DISTINCT value FROM seq_region_attrib "
+                        "JOIN attrib_type USING (attrib_type_id) "
+                        "WHERE attrib_type.code='genome_component';"
+                    ),
+                ).split("\n")
+                if component
+            ]
+
+            # Generate dump filenames
+            dump_filenames = (
+                [
+                    (f"{fasta_file_name}_{component}", component)
+                    for component in genome_components
+                ]
+                if genome_components
+                else [(fasta_file_name, "")]
+            )
+
+            # Collect download information
+            for filename, genome_component in dump_filenames:
+                logging.info("fasta_file_name=%s", filename)
+                logging.info("genome_component=%s", genome_component)
+                download_content.append({
+                    "host": host,
+                    "port": port,
+                    "core_db": core_db,
+                    "genome_component": genome_component,
+                    "fasta_filename": filename,
+                })
+
+    return download_content
+
+def main():
+    """
+    Main function to parse arguments and handle the processing of a YAML file to dump a list of FASTA files.
+    """
+    setup_logging()
```

Comment:
Could the basic logging config be done here?
```suggestion
    logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
```